.github/workflows/release-staging.yml

```diff
@@ -7,6 +7,7 @@ name: Release Staging
 
 env:
   cache-name: staging-cd
+  publicecr-name: x5x8k3n7
```

Comment:
Should we be able to put the alias here when it's approved?

Reply:
Yes! No idea when that could happen though, Amazon has only rolled out Public ECR two weeks ago so there isn't much information. Hopefully, it'll be done by the time we all come back from the holidays.